src/Mono.Android/map.csv

```diff
@@ -64,6 +64,8 @@
 0,,,android/view/Surface.FX_SURFACE_MASK,98304
 0,,,android/view/Surface.FX_SURFACE_NORMAL,0
 
+// Manually added because cannot be auto generated.
```

Comment:
I would probably change this to something like `// Manually added to preserve backwards compatibility (leave the original const as int)` so we better understand why we're doing this.

Reply:
Sure! fixed